recipes/configs/dev/8B_full_experimental.yaml

```diff
@@ -57,7 +58,7 @@ loss:
   _component_: torchtune.modules.loss.CEWithChunkedOutputLoss
 max_steps_per_epoch: null
 gradient_accumulation_steps: 1
-
+compile: False
```

Comment:
Should we be explicit that this will `torch.compile` the model and loss? `compile` seems like a vague name

Reply:
agreed. I think we can add a comment, similar to packed

"compile=False  # Set to true for perf/memory improvement"

wdyt?

Reply:
Add some comment there?

Reply:
Ok, will add